build.gradle.kts

```diff
@@ -173,6 +173,23 @@ subprojects {
     }
   }
 
+  apply(plugin = "com.diffplug.spotless")
```

Comment:
We should be loading the config from `.editorconfig`, rather than partially duplicating it here. As-is the two disagree on where `java.*` imports go.

Reply:
But it seems haven't support yet. See https://github.com/diffplug/spotless/issues/142